### PR TITLE
Add slovak translation for Url and Length validator

### PR DESCRIPTION
**Pull request by pulzarraider**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.sk.xlf [hunk 1]
```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>Súbor nesmie byť prázdny.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Hostiteľa nebolo možné rozpoznať.</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Táto hodnota nezodpovedá očakávanej znakovej sade {{ charset }}.</target>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.sk.xlf]: Google Translate says that `nezodpovedá` should be `zodpovedá` Is that correct?

Comment (pulzarraider) [inline src/Symfony/Component/Validator/Resources/translations/validators.sk.xlf]: Google translate is wrong.
`zodpovedá` == `match`
`nezodpovedá` == `does not match`. It's teh negative of `zodpovedá`.

I've already contribute a fix to Google translate.

Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.sk.xlf]: Great! Thank you.

**Comment by javiereguiluz** [comment]
:+1:

Status: reviewed

**Comment by fabpot** [comment]
Thank you @pulzarraider.

**Pull request merged**